bin/webindex

```diff
@@ -0,0 +1,114 @@
+  exit 1
+fi
+if [ -z $FLUO_HOME ]; then
+  echo "FLUO_HOME must be set in bash env or conf/webindex-env.sh"
+  exit 1
+fi
+if [ ! -d $FLUO_HOME ]; then
+  echo "FLUO_HOME=$FLUO_HOME does not exist"
+  exit 1
+fi
+
+COMMAND_LOGFILE=$WI_HOME/logs/$1_`date +%s`.log
+
+case "$1" in
+copy)
+  . $BIN_DIR/impl/base.sh "${*:2}"
+  spark-submit --class io.fluo.webindex.data.Copy \
+    --master yarn-client \
+    --num-executors `get_prop sparkExecutorInstances` \
+    --executor-memory `get_prop sparkExecutorMemory` \
+    --conf spark.shuffle.service.enabled=true \
+    $WI_DATA_DEP_JAR $WI_HOME/conf/data.yml &> $COMMAND_LOGFILE &
+  echo "Started copy.  Logs are being output to $COMMAND_LOGFILE"
+	;;
+init)
+  $BIN_DIR/impl/init.sh &> $COMMAND_LOGFILE &
+  echo "Started initialization.  Logs are being output to $COMMAND_LOGFILE"
+	;;
+load)
+  . $BIN_DIR/impl/base.sh "${*:2}"
+  FLUO_APP=`get_prop fluoApp`
+  FLUO_PROPS=$FLUO_HOME/apps/$FLUO_APP/conf/fluo.properties
+  if [ ! -f $FLUO_PROPS ]; then
+    echo "Fluo properties file must exist at $FLUO_PROPS"
+    exit 1
+  fi
+  spark-submit --class io.fluo.webindex.data.Load \
+    --master yarn-client \
+    --num-executors `get_prop sparkExecutorInstances` \
+    --executor-memory `get_prop sparkExecutorMemory` \
+    --conf spark.shuffle.service.enabled=true \
+    --files $FLUO_PROPS \
+    $WI_DATA_DEP_JAR $WI_HOME/conf/data.yml &> $COMMAND_LOGFILE &
+  echo "Started load.  Logs are being output to $COMMAND_LOGFILE"
+	;;
+reindex)
+  . $BIN_DIR/impl/base.sh "${*:2}"
+  spark-submit --class io.fluo.webindex.data.Reindex \
+    --master yarn-client \
+    --num-executors `get_prop sparkExecutorInstances` \
+    --executor-memory `get_prop sparkExecutorMemory` \
+    $WI_DATA_DEP_JAR $WI_HOME/conf/data.yml &> $COMMAND_LOGFILE &
+  echo "Started reindex.  Logs are being output to $COMMAND_LOGFILE"
+	;;
+ui)
+  cd $WI_HOME/modules/ui
+  mvn clean install -DskipTests
+  DROPWIZARD_CONFIG=""
+  if [ -f $WI_HOME/conf/dropwizard.yml ]; then
+    DROPWIZARD_CONFIG=$WI_HOME/conf/dropwizard.yml
+    echo "Running with dropwizard config at $DROPWIZARD_CONFIG"
+  fi
+  java -jar $WI_HOME/modules/ui/target/webindex-ui-0.0.1-SNAPSHOT.jar server $DROPWIZARD_CONFIG
+  ;;
+splits)
+  . $BIN_DIR/impl/base.sh "${*:2}"
+  spark-submit --class io.fluo.webindex.data.CalcSplits \
+    --master yarn-client \
+    --num-executors `get_prop sparkExecutorInstances` \
+    --executor-memory `get_prop sparkExecutorMemory` \
+    --conf spark.shuffle.service.enabled=true \
+    $WI_DATA_DEP_JAR $WI_HOME/conf/data.yml &> $COMMAND_LOGFILE &
+  echo "Started splits calculation.  Logs are being output to $COMMAND_LOGFILE"
+  ;;
+*)
+  echo -e "Usage: webindex <command> (<argument>)\n"
+  echo -e "Possible commands:\n"
+  echo "  copy        Copies CommonCrawl data files from AWS into HDFS init & load directories"
+  echo "  init        Starts Spark job that initializes Fluo & Accumulo with data in HDFS init directory "
```

Comment:
could change `initializes Fluo & Accumulo with` to `initializes Fluo & Accumulo tables using`